relay-server/src/services/buffer/envelope_store/sqlite.rs

```diff
@@ -24,35 +25,78 @@ use sqlx::{Pool, QueryBuilder, Row, Sqlite};
 use tokio::fs::DirBuilder;
 use tokio::time::sleep;
 
+const ZSTD_MAGIC_WORD: &[u8] = &[40, 181, 47, 253];
+
 /// Struct that contains all the fields of an [`Envelope`] that are mapped to the database columns.
+#[derive(Clone, Debug)]
 pub struct InsertEnvelope {
```

Comment:
Since we now use this type not only for insertion, I would rename it:
```suggestion
pub struct SerializedEnvelope {
```

Reply:
I concur. `DatabaseEnvelope` might also work.